meetings/2023-02-02.md

```diff
@@ -0,0 +1,210 @@
+* eP: Slight preference for merge and then track with what the diffs are. Benefit for that kind of workflow.
+* SC: Agree.
+* AC: +1 merge
+* RG: Fine. We're going to fix it any way.
+
+ACTION: SC to merge.
+
+ACTION: SC to follow-up on creating issue for `filter`
+
+ACTION: SC to do editorial update to clarify the diffs better.
+
+### initial StreamingHTTPChannel2023
+URL: https://github.com/solid/notifications/pull/139
+
+* SC: Okay to merge?
+* AC: Good.
+* RG: Okay.
+* eP: Prefer to merge. Bikeshed allows partials. I want to do one common paragraph for the Webhook terminology. It can be useful for WebPush too. Have a paragraph that can be used for all.
+
+ACTION: eP to merge and follow up with reusable paragraph on terminology.
+
+### Add EventSourceChannel2023 Notification Type
+URL: https://github.com/solid/notifications/pull/144
+
+* SC: Okay to merge?
+* AC: Okay.
+
+ACTION: AC to merge and follow up with reusable paragraph on terminology.
+
+### initial WebhookChannel2023
+URL: https://github.com/solid/notifications/pull/146
+
+* SC: Okay to merge or review?
+* eP: Changed from draft to review recently. It can wait a bit.
+
+ACTION: eP will reach out to Jackson to edit / follow up.
+
+
+### Rename Subscription Resource to Subscription Service?
+URL: https://github.com/solid/notifications/issues/63
+
+* SC: There is some support to use "subscription service" or "subscription service resource".
+* SC: This change doesn't affect conformance, if we agree on change, I suggest that we quickly roll it into Version 0.2.0, TR/notifications-protocol-20221231.
+* SC: Any objections or concerns to change to "subscription service"?
+* AC: No strong opinion.
+* eP: Okay to rename to subscription service.
+* eP: Looking at the draft in push API, it uses subscription resource but a bit different ( in https://www.rfc-editor.org/rfc/rfc8030#section-2 ). Would be good to use another term to not potentially overlap.
+* SC: We have some pluses and no objections. We can do it.
+* eP: Justified to change terminology for 0.2.0.
+* AC: FIne with me. Aligning on terminology should be done earlier than later. Adding/changing functionality should be done more carefully.
+* RG: Yes.
+    
+
+### Rename Subscription Client
+URL: https://github.com/solid/notifications/issues/142
+
+* SC: I didn't feel it is necessary and Christoph seemed to prefer subscription client as well.
+* AC: No strong opinion.
+* eP: DiscoveryClient doesn't make too much sense. In case there is a subscription, it is the notification receiver that's doing the discovery. We only need to adjust which part has to implement the discovery. if there is no subscription, then it is just the notification receiver that does it. I'd wait for it on this and not for 0.2.0. justify in the discovery that sometimes the subscription client does it and sometimes the notification receiver - only channels that have `receiveFrom` and so no need for subscription client.
```

Comment:
```suggestion
* eP: DiscoveryClient doesn't make too much sense. In case there is a subscription, it is the notification receiver that's doing the discovery. We only need to adjust which part has to implement the discovery. if there is no subscription, then it is just the notification receiver that does it. I'd wait for it on this and not for 0.2.0. Justify in the discovery that sometimes the subscription client does it, and sometimes the notification receiver — only channels that have `receiveFrom`, and so no need for subscription client.
```